### ISSUE TITLE
[Select] triggerProps aria-labelledby is missing id prefix

### ISSUE THREAD

**Issue by visualjerk** [chakra-ui/zag#444]
# 🐛 Bug report

For select api, the `aria-labelledby` on `api.triggerProps` is missing id prefix and therefore not set to a valid `id`.

## 💥 Steps to reproduce

1. Create a select api with id "select-id"
1. Use api.triggerProps on an element
1. Notice that `aria-labelledby` is set to `label`

## 💻 Link to reproduction

CodeSandbox reproduction: https://codesandbox.io/s/determined-browser-sf9dh2?file=/src/App.vue

## 🧐 Expected behavior

1. `aria-labelledby` is set to the id from `api.labelProps` (in the above case `select-id:label`)

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Zag Version      | 0.1.2  |
| Browser          |  Chrome 107.0.5304.121 |
| Operating System |  MacOS  |
